spec/controllers/dashboard/firms_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+module Dashboard
+  RSpec.describe Dashboard::FirmsController, type: :controller do
+    let(:principal) { FactoryGirl.create(:principal) }
```

Comment:
I'd like us to agree on a standard for this (or agree to have no standard). It's not a good idea to have style conversations repeatedly. 

If it's a blocker for this PR, let's decide it here and store it somewhere that we can refer to. If not, let's discuss it when we're in the office this week (it's dev chat this week I think?)